jupyter/minimal/ubi8-python-3.8/Dockerfile

```
@@ -25,4 +25,7 @@ RUN chmod -R g+w /opt/app-root/lib/python3.8/site-packages && \
 
 WORKDIR /opt/app-root/src
 
+RUN sed -i -e "s/Python.*/$(python --version)\",/" /opt/app-root/share/jupyter/kernels/python3/kernel.json
```

Break this line to improve readability:

```suggestion
RUN sed -i -e "s/Python.*/$(python --version)\",/" \
    /opt/app-root/share/jupyter/kernels/python3/kernel.json
```